domain/personallearningplan/src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/domain/CommonBuilders.kt

```diff
@@ -7,7 +7,7 @@ import java.util.UUID
  * prison numbers, times and dates etc.
  */
 
-fun aValidPrisonNumber() = "A1234BC"
+fun randomValidPrisonNumber() = "A1234BC"
```

Comment:
One for a future change to make the impl return a valid one? (its not very random at the moment 🤣 )